distribution/docs/src/main/resources/_contents/_developing/developing-intro-contents.adoc

```diff
@@ -0,0 +1,816 @@
+
+${branding} as a framework hosts ${branding} applications, which are extensible by adding components via OSGi.
+The best example of this is the ${branding} Catalog (API), which offers extensibility via several types of Catalog Components.
+The ${branding} Catalog API serves as the foundation for several applications and resides in the applications tier.
+
+The Catalog Components consist of Endpoints, Plugins, Catalog Frameworks, Sources, and Catalog Providers.
+Customized components can be added to ${branding}.
+
+==== Nomenclature
+
+Capability:: A general term used to refer to an ability of the system
+Application:: One or more features that together form a cohesive collection of capabilities
+Component:: Represents a portion of an Application that can be extended
+Bundle:: Java Archives (JARs) with special OSGi manifest entries.
+Feature:: One or more bundles that form an installable unit;  Defined by Apache Karaf but portable to other OSGi containers.
+
+==== OSGi Core
+
+${branding} makes use of OSGi v4.2 to provide several capabilities:
+
+* Has a Microkernel-based foundation, which is lightweight due to its origin in embedded systems.
+* Enables integrators to easily customize components to run on their system.
+* Software applications are deployed as OSGi components, or bundles.
+Bundles are modules that can be deployed into the OSGi container (Eclipse Equinox OSGi Framework by default).
+* Bundles provide flexibility allowing integrators to choose the bundles that meet their mission needs.
+* Bundles provide reusable modules that can be dropped in any container.
+* Provides modularity, module-based security, and low-level services, such as  Hypertext Transfer Protocol (HTTP), logging, events (basic publish/subscribe), and dependency injection.
+* Implements a dynamic component model that allows application updates without downtime.
+Components can be added or updated in a running system.
+* Standardized Application Configuration (ConfigurationAdmin and MetaType)
+
+OSGi is not an acronym, but if more context is desired the name Open Specifications Group Initiative has been suggested.
+
+More information on OSGi is available at http://www.osgi.org/.
+
+==== Built on Apache Karaf
+
+Apache Karaf is a FOSS product that includes an OSGi framework and adds extra functionality, including:
+
+${admin-console}:: Useful for configuring applications, installing/uninstalling features, and viewing services such as metrics.
+${command-console}:: Provides command line administration of the OSGi container. All functionality in the ${command-console} can also be performed via this command line console.
+Logging:: Provides centralized logging to `data/log/${branding-lowercase}.log`. Security logging is provided at `data/log/security.log`. Ingest error logging can be viewed in `data/log/ingest_error.log`.
+Provisioning:: Of libraries or applications.
```

Comment:
This is another one where I honestly think that is a zero instead of a capital 'O'.